nettacker/database/db.py

```
@@ -290,9 +589,40 @@ def get_scan_result(id):
+                connection.close()
+            except Exception:
+                pass
+    else:
+        report = session.query(Report).filter_by(id=id).first()
+        if not report:
+            return None
+
+        try:
+            return report.report_path_filename, open(str(report.report_path_filename), "rb").read()
+        except IOError as e:
+            logger.error(f"Failed to read report file: {e}")
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)